tools/perf/dashboard_auth.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+"""A tool to perform authorization to brave-perf-dashboard.appspot.com"""
+
+import sys
+import json
+import io
+import os
+import argparse
+import logging
+
+from typing import Optional
+
+# pylint: disable=import-error
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+
+SERVICE_ACCOUNT_FILE_ENV_NAME = 'PERF_DASHBOARD_SERVICE_ACCOUNT_FILE'
```

Comment:
@fmarier @diracdeltas 
Previously we have a discussion about storing credentials/token for submitting info to the dashboard.
The current approach (implemented here):
1. the service account secret files stores in 1Password.
2. CI passed it here as a secret file by setting `PERF_DASHBOARD_SERVICE_ACCOUNT_FILE`
3. This code doesn't store anything, it created a token from the secret file each time it needed (a few times perf CI run);

The devops PR: https://github.com/brave/devops/pull/9097
Does it look okay to you?

Reply:
Looks good :+1: